app/controllers/api/v1/sessions_controller.rb

```diff
@@ -17,11 +17,14 @@ def session_params
 
   def create_params
     current_time = Time.now.utc
-    { session_type: session_params[:type],
+    {
+      session_type: session_params[:type],
       user: current_user,
       status: 0,
       start_time: current_time,
-      last_ping_time: current_time }
+      end_time: current_time,
```

Comment:
@nronas why are we updating the end_time here ? Shouldn't it be nil just after creation ?

Reply:
that was based on my premise that i had that end_time should be the source of truth. Will amend now that we have discuss to move forwards with the last_ping_time